backend/bufftracer/package.json

```diff
@@ -49,8 +49,8 @@
     "typescript": "^4.4.3"
   },
   "dependencies": {
-    "@bufferapp/bufflog": "0.3.1",
-    "dd-trace": "1.4.1",
+    "@bufferapp/bufflog": "0.5.1",
+    "dd-trace": "^5.18.0",
```

Comment:
Hey @dinostheo, I believe we cannot do this upgrade for now because Core services are still running on Node 14. 

It is mentioned in the changelog [at version 4](https://github.com/DataDog/dd-trace-js/blob/master/MIGRATING.md#30-to-40).

For the other breaking changes, I think we are good. For the new env variables, please make sure all services in buffer-mono does have it as well 😊 

The infra team has the goal to migrated Core services to node 20 this quarter, so you will want to coordinate with them.

Reply:
Thanks @philippemiguet, I didn't consider that we were still using v14.